socorro/external/postgresql/models.py

```diff
@@ -14,11 +14,23 @@
 from sqlalchemy.orm import relationship
 import sqlalchemy.types as types
 
-try:
-    from sqlalchemy.dialects.postgresql import *
-    from sqlalchemy.dialects.postgresql.base import ischema_names
-except ImportError:
-    from sqlalchemy.databases.postgres import *
+from sqlalchemy.dialects.postgresql import (
+    INTERVAL,
+    INTEGER,
+    TEXT,
+    TIMESTAMP,
+    NUMERIC,
+    REAL,
+    DATE,
+    BOOLEAN,
+    UUID,
+    VARCHAR,
+    ARRAY,
+    BIGINT,
+    SMALLINT,
+    CHAR,
+)
+from sqlalchemy.dialects.postgresql.base import ischema_names
```

Comment:
The try/except ImportError must date back to ancient days. Also, I'm pretty confident this file was generated code and not written code. In hindsight, I should probably just have put it into `setup.cfg` or something as a file to ignore. However, we don't have a global solution to flake8 yet.